diff --git a/src/modules/Modules.cpp b/src/modules/Modules.cpp
index e477574dd..65396d4e7 100644
--- a/src/modules/Modules.cpp
+++ b/src/modules/Modules.cpp
@@ -9,6 +9,7 @@
 #include "input/UpDownInterruptImpl1.h"
 #include "input/i2cButton.h"
 #include "modules/SystemCommandsModule.h"
+#include "SignalReplyModule.h"
 #if HAS_TRACKBALL
 #include "input/TrackballInterruptImpl1.h"
 #endif
@@ -119,6 +120,7 @@ void setupModules()
         buzzerFeedbackThread = new BuzzerFeedbackThread();
     }
 #endif
+new SignalReplyModule();
 #if !MESHTASTIC_EXCLUDE_ADMIN
     adminModule = new AdminModule();
 #endif
